lib/cli-create-embed-token.ts

```diff
@@ -0,0 +1,54 @@
+import * as powerbi from 'powerbi-api';
+import * as msrest from 'ms-rest';
+import {Cli as cli} from './cli';
+import {Config as config} from './config';
+import * as program from 'commander';
+
+export default function CliCreateEmbedToken() {
+    let pkg = require('../package.json');
+
+    function list(val) {
+        return val.split(',');
+    }
+
+    program.version(pkg.version)
+        .option('-c, --collection <collection>', 'The Power BI workspace collection')
+        .option('-k, --accessKey <accessKey>', 'The Power BI workspace collection access key')
+        .option('-w, --workspace <workspaceId>', 'The Power BI workspace id')
+        .option('-r, --report <reportId>', 'The Power BI report id')
+        .option('-u, --username [username]', 'The RLS username')
+        .option('--roles [roles]', 'The RLS roles', list);
```

Comment:
I think you should explicitly state in the help and example that this accepts a comma separated list of roles.